c7n/resources/rdscluster.py

```diff
@@ -54,27 +54,43 @@ class Meta(object):
 class Delete(BaseAction):
 
     schema = type_schema(
-        'delete', **{'skip-snapshot': {'type': 'boolean'}})
+        'delete', **{'skip-snapshot': {'type': 'boolean'},
+                     'delete-instances': {'type': 'boolean'}})
 
-    def process(self, resources):
-        self.skip = self.data.get('skip-snapshot', False)
+    def process(self, clusters):
+        skip = self.data.get('skip-snapshot', False)
+        delete_instances = self.data.get('delete-instances', False)
```

Comment:
should this default to True, it feels wierd to delete the cluster but leave all the instances bits (compared to an asg that left all its instances?) is this even nesc? do individual instances survive the cluster?